duckduckgo/Android
# ADS: Widgets migration

Task/Issue URL: 

### Description

Migrate widgets to new styles

### Steps to test this PR

- Install from this branch
- Add widget to your home screen
- Check widgets match new designs

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20221230_163631_One UI Home](https://user-images.githubusercontent.com/20798495/210092677-96c4eaa8-4008-48d3-8ec2-25752184ed06.jpg)|![Screenshot_20221230_163636_One UI Home](https://user-images.githubusercontent.com/20798495/210092673-46145d5d-170c-4edc-8ccf-d5b7885f9591.jpg)|
